src/main/resources/settings.yaml

```diff
@@ -1,11 +1,12 @@
 port: ${FHIRSPARK_PORT:-3001}
-fhirDbBase: ${FHIRSPARK_FHIRBASE:-http://localhost:8080/fhir}
+fhirDbBase: ${FHIRSPARK_FHIRBASE:-http://localhost:8080/fhir/}
```

Comment:
I intentionally removed the `/` because otherwise some FHIR servers like blaze might return a `308 Permanently moved` status code.